tecken/download/views.py

```diff
@@ -151,7 +151,7 @@ def download_symbol(request, symbol, debugid, filename, try_symbols=False):
             # from the host.
             if (
                 settings.DEBUG
-                and StorageBucket.URL_FINGERPRINT["emulated-s3"] in url
+                and StorageBucket(url).backend == "emulated-s3"
                 and "http://minio:9000" in url
                 and request.get_host() == "localhost:8000"
             ):  # pragma: no cover
```

Comment:
Is the issue that this code is solving that inside the web container, the minio container has one name and from the docker host, it has another. Is that right?

Reply:
That's the way I read it too. Minio may have a configuration that will make this go away, or we can do the nip.io trick, or maybe we hold our nose until the S3 code goes away.

Reply:
I'm +1 on whatever you want to do. Given our current status, I wouldn't hold this PR up for "the best solution to all things".

Reply:
I'm adding it to my list, but not changing for this PR.